core/block.js

```diff
@@ -193,6 +193,13 @@ Blockly.Block.prototype.data = null;
 Blockly.Block.prototype.colour_ = '#000000';
 
 /**
+ * Colour of the block as HSV hue value (0-360)
+ * @type {number}
```

Comment:
Because the color may not have been derived from the hue, initialize this to null.  This also means the @type is nullable: `{?number}`.